src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.product {
+  width: $card-Width;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  float: left;
```

Comment:
1. it's a bem block
check the check list
<img width="996" alt="image" src="https://user-images.githubusercontent.com/102411130/200261462-4b3b561a-94ba-44f7-a49a-85b8bd2327da.png">

2. remove `font-family` to `html` tag and `box-sizing` to `*` tag

3. `400` if a default `font-weight`

4. check css property order
https://codeguide.academy/html-css.html#css-order

check all such cases